src/http/content.cr

```diff
@@ -134,7 +143,7 @@ module HTTP
 
     private def read_chunk_start
       read_chunk_end
-      @chunk_remaining = @io.gets.not_nil!.to_i(16)
+      read_chunk_remaining
```

Comment:
I just remembered that we have `read_line` that raises an `IO::EOFError`. Wouldn't it be enough to deal with this?